javaparser-core-testing/src/test/java/com/github/javaparser/printer/XmlPrinterTest.java

```diff
@@ -25,38 +25,73 @@
 import org.junit.jupiter.api.Test;
 
 import static com.github.javaparser.StaticJavaParser.parseExpression;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
 
 class XmlPrinterTest {
+
+    DocumentBuilderFactory dbf;
+    DocumentBuilder db;
+    {
+        try {
+            dbf = DocumentBuilderFactory.newInstance();
+            dbf.setNamespaceAware(true);
+            dbf.setCoalescing(true);
+            dbf.setIgnoringElementContentWhitespace(true);
+            dbf.setIgnoringComments(true);
+            db = dbf.newDocumentBuilder();
+        } catch (ParserConfigurationException ex) {
+            throw new RuntimeException(ex);
+        }
+    }
+
+    public Document getDocument(String xml) throws SAXException, IOException {
```

Comment:
Is there no need to close the inputStream?

Reply:
As per [JavaDoc](https://docs.oracle.com/javase/8/docs/api/java/io/ByteArrayInputStream.html#close--), closing a ByteArrayInputStream has no effect.